pyiron_base/archiving/import_archive.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pandas 
+import numpy as np
+from shutil import copytree
+#from pyiron_base import Project
+
+def getdir(path): 
+    path_base_name = os.path.basename(path)
+    if path_base_name == "":
+        return os.path.basename(os.path.dirname(path))
+    else: 
+        return path_base_name
+
+def update_id_lst(record_lst, job_id_lst):
+    masterid_lst = []
+    for masterid in record_lst:
+        if np.isnan(masterid):
+            masterid_lst.append(masterid)
+        elif isinstance(masterid, int) or isinstance(masterid, float):
+            masterid = int(masterid)
+            masterid_lst.append(job_id_lst[masterid])
+    return masterid_lst
+
+def import_jobs(project_instance, directory_to_import_to, archive_directory, df):
+    # Copy HDF5 files
+    archive_name = getdir(path=archive_directory)
+    cwd = os.getcwd()
+    directory_to_import_to = directory_to_import_to.split("/")[1]
```

Comment:
Use `os.path.dirname` and `os.path.abspath` to make things a bit clearer and shorter.

Reply:
Done!

Reply:
Great!